### PR TITLE
Adding separators between ldap, username, email, github

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Also fixing a minor issue with code themes.

### CODE REVIEW

**Comment by it33** [comment]
Fixed approved in triage, next step is code review, 

Thanks @asaadmahmoodspin!

**Comment by jwilander** [comment]
+1

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
@asaadmahmoodspin Can you squash your commits please?

File: docker/2.0/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ VOLUME /var/lib/mysql
 WORKDIR /mattermost
 
 # Copy over files
-ADD https://github.com/mattermost/platform/releases/download/v2.0.0-rc2/mattermost.tar.gz /
+ADD https://github.com/mattermost/platform/releases/download/v2.0.0/mattermost.tar.gz /
```
Comment (asaadmahmoodspin) [inline docker/2.0/Dockerfile]: Hey @crspeller I don't remember making this change, is this okay? I think this got added from another commit.

**Comment by asaadmahmoodspin** [comment]
Merged the commits.

File: docker/2.0/Dockerfile (hunk 1 above)
Comment (crspeller) [inline docker/2.0/Dockerfile]: That's something from master. Its fine to change. Just make sure nothing else not in and let me know.

**Comment by asaadmahmoodspin** [comment]
Yea then that's pretty much it.

**Pull request merged**